examples/todo-list/frontend/src/backend-sdk/remote.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I think this should be removed. Isn't it already in .gitignore? If not, can you please add it?